### PR TITLE
Fix elevation-1 variable

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

**Comment by jespino** [comment]
@ogi-m this should solve the problem of focalboard overriding the elevation-1 variable with a different value. It is used in mattermost in the :hover of the .post-image__column class, and in the `.post-menu` of the `:hover` post. I don't think testing this is really needed, but following the rule stablished about merging things in 0.9, I want your approval before merge.

**Review by asaadmahmood — APPROVED** [review]

**Review by ogi-m — APPROVED** [review]
Thanks @jespino, LGTM

**Pull request merged**
